packages/wp-build/src/build.mjs

```diff
@@ -52,6 +52,7 @@ const IGNORE_PATTERNS = [
 	'**/*.native.*',
 	'**/*.ios.*',
 	'**/*.android.*',
+	'**/*.{spec,test}.*',
```

Comment:
It might be good to sync "spec" variations into [the entry above aimed at test folders](https://github.com/WordPress/gutenberg/blob/00c8b38b595510c4b6b9dd56484cfdbda442ff75/packages/wp-build/src/build.mjs#L50), since I've also seen `spec` folders.